packages/cli/src/lib/bundler/config.ts

```diff
@@ -124,17 +137,94 @@ export function createConfig(
         ? '[name].chunk.js'
         : '[name].[chunkhash:8].chunk.js',
     },
-    optimization: optimization(options),
     plugins,
+  };
+}
+
+export function createBackendConfig(
+  paths: BundlingPaths,
+  options: BackendBundlingOptions,
+): webpack.Configuration {
+  const { checksEnabled, isDev } = options;
+
+  const { loaders } = transforms(options);
+
+  return {
+    mode: isDev ? 'development' : 'production',
+    profile: false,
+    ...(isDev
+      ? {
+          watch: true,
+          watchOptions: {
+            ignored: [/node_modules\/(?!\@backstage)/],
+          },
+        }
+      : {}),
+    externals: [
+      nodeExternals({
```

Comment:
Did feel we ever concluded the discussion about these? 😅 

What's the performance impact of not doing this?

Afaik we'd also want to list the module dirs of all adjacent packages, as we're missing those right now.

Reply:
Ah just saw the todo in the PR body :grin: makes sense to investigate this later. Although if we don't end up removing these we should figure out how to list all ze node_modules, but probably not worth doing that up-front

Reply:
Some modules just failing, will try it out as a separate effort ;)